File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.swift
class Solution {
    func kidsWithCandies(_ candies: [Int], _ extraCandies: Int) -> [Bool] {
        var result = [Bool]()
        var max = candies.max()!

        for i in 0..<candies.count {
            let a = candies[i] + extraCandies
            if a >= max {
                result.append(true)
            } else {
                result.append(false)
            }
        }
        return result
    }
}

File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        max_value = max(candies)
        result = []
        for i in candies:
            if i + extraCandies >= max_value:
                result.append(True)
            else: 
                result.append(False)
        
        return result